installer/scripts/omsadmin.sh

```diff
@@ -352,6 +367,25 @@ set_proxy_setting()
     fi
 }
 
+set_omsagent_proc_limit()
```

Comment:
Why? And if I add it, what should the minimum be? If a customer is setting this limit, they already are familiar with how processes work, and know that setting the limit too low may hinder functionality.